src/GameServer.pas

```diff
@@ -768,7 +768,7 @@ procedure StartGame;
       begin
         Government := gDespotism;
         Money := StartMoney;
-        TaxRate := 30;
+        TaxRate := 0;
```

Comment:
"Newbies" can learn how to increase tax then (cevo its not a game for a mass, every one figure it out in second game that 100 res at start is best)
50 starting gold? hmm.. it may speed up some starting things (finishing settlers / town guards) or be a reserve for later. I would try play game with starting cash.